docs/web/sdk-service-injection.mdx

```diff
@@ -1,13 +1,29 @@
 ---
-title: Injecting Services
+title: Directly Referencing Services
```

Comment:
If we don't like the `injecting` title, then perhaps we could call this `Referencing Other Services`? or similar?

We'd likely want to update the equivalent in the components section as well if we change.

Reply:
See above comment

Reply:
Changed to "Service Injection"